v2/internal/frontend/desktop/windows/win32/theme.go

```diff
@@ -50,6 +59,16 @@ func SetTheme(hwnd uintptr, useDarkMode bool) {
 	}
 }
 
+func EnableTranslucency(hwnd uintptr, backdrop BackdropType) {
+	fmt.Printf("%+v\n", windowsVersion)
+	if IsWindowsVersionAtLeast(10, 0, 22579) {
+		dwmSetWindowAttribute(hwnd, DwmwaSystemBackdropType, unsafe.Pointer(&backdrop), unsafe.Sizeof(&backdrop))
+	} else {
+		println("NOOOOOO here!!!!MICA")
```

Comment:
Should we print out a more detailed message, like "Translucency not supported on Windows versions before 10.0.22579"?

Reply:
hahah! Indeed! 😂